packages/react-core/src/demos/PrimaryDetail.md

```diff
@@ -978,7 +978,7 @@ class PrimaryDetailContentPadding extends React.Component {
 
     const drawerContent = (
       <React.Fragment>
-        <Toolbar id="content-padding-data-toolbar" className="pf-m-toggle-group-container">
+        <Toolbar id="content-padding-data-toolbar" className="pf-m-page-insets">
```

Comment:
Would the `inset` prop that was added to Toolbar do what we want here?  It was merged in PR #4757

Reply:
I don't think so. We haven't added a `hasPageInsets prop to add `styles.modifiers.pageInsets = 'pf-m-page-insets'` yet.